### PR TITLE
Add Marketplace doc to 4.4

### PR DESCRIPTION
**Pull request by robszumski**
Add s a simple doc about Red Hat Marketplace that is mostly links out to external content at marketplace.redhat.com. This location is currently behind a whitelist but I can show you that content if required.

I am happy to make changes or have someone take over this PR if that is easier.

cc: @maxwelldb

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@sferich888 can you confirm that this content is ok, especially the links?

@maxwelldb what is the status with this? Does this need to be in for 4.4 GA?

**Comment by maxwelldb** [comment]
@vikram-redhat I'm modularizing and editing as time allows for versions 4.4 and beyond. It is not promised for 4.4 GA, no.

**Comment by maxwelldb** [comment]
@robszumski Restructured this for FCC and did a little bit of editing. Thoughts?

File: modules/red-hat-marketplace-features.adoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+[id="red-hat-marketplace-features_{context}"]
+= Red Hat Marketplace features
+
+Cluster administrators can use link:https://marketplace.redhat.com/en-us/documentation/getting-started[Red Hat Marketplace] to manage software on {product-title}, give developers self-service access to deploy application instances, and correlate application usage against a quota.
+
+[id="marketplace-clusters_{context}"]
+== Connect {product-title} clusters to the Marketplace
+
+You can install a common set of applications on {product-title} clusters that you connect to the Marketplace. You can also use the Marketplace to track cluster usage against your subscriptions or quota. Additional users can be added
+// (NOTE: added to what?)
```
Comment (robszumski) [inline modules/red-hat-marketplace-features.adoc:14]: to the account to view Marketplace apps purchased, installed, etc

**Review by robszumski — COMMENTED** [review]
LGTM. Added one note about your todo

File: modules/red-hat-marketplace-features.adoc (hunk 1 above)
Comment (maxwelldb) [reply modules/red-hat-marketplace-features.adoc:14]: @robszumski Do you think calling this out in the OCP docs is important, or could it be removed?

Comment (robszumski) [reply modules/red-hat-marketplace-features.adoc:14]: it's a main use-case that helps you understand the product, so I would say yet it should stay

Comment (maxwelldb) [reply modules/red-hat-marketplace-features.adoc:14]: Sure. Wasn't positive if user management was an 'expected to be there' sort of functionality or not. @robszumski

**Comment by maxwelldb** [comment]
@robszumski Last thing: Is it important to call out a "Procurement department" specifically?

**Comment by robszumski** [comment]
@maxwelldb This was one of the use-cases, but it doesn't have to be called out that specifically.

**Comment by maxwelldb** [comment]
@robszumski Any concerns with this content as it is, now? If not, could you squash it to a single commit, force push, and then I'll get it into peer review? 

We're holding off on any merges to master or release branches until 4.4 GA, so no rush.

**Comment by robszumski** [comment]
LGTM, squashed

**Comment by maxwelldb** [comment]
@openshift/team-documentation PTAL at this. Ali and QE are both fine with it.

File: applications/red-hat-marketplace.adoc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[id="red-hat-marketplace"]
+= The Red Hat Marketplace
+include::modules/common-attributes.adoc[]
+:context: red-hat-marketplace
+
+toc::[]
+
+The link:https://marketplace.redhat.com[Red Hat® Marketplace] is an open cloud marketplace that makes it easy to discover and access certified software for container-based environments that run on public clouds and on-premesis.
```
Comment (bergerhoffer) [inline applications/red-hat-marketplace.adoc:8]: Do we need the r-ball after Red Hat here? I don't think we do that anywhere else that I'm aware of

File: modules/red-hat-marketplace-features.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+[id="red-hat-marketplace-features_{context}"]
+= Red Hat Marketplace features
+
+Cluster administrators can use link:https://marketplace.redhat.com/en-us/documentation/getting-started[Red Hat Marketplace] to manage software on {product-title}, give developers self-service access to deploy application instances, and correlate application usage against a quota.
+
+[id="marketplace-clusters_{context}"]
+== Connect {product-title} clusters to the Marketplace
+
+You can install a common set of applications on {product-title} clusters that you connect to the Marketplace. You can also use the Marketplace to track cluster usage against your subscriptions or quota. Users that you add by using the Marketplace have their product usage tracked and billed to your organization.
```
Comment (bergerhoffer) [inline modules/red-hat-marketplace-features.adoc:13]: This is the only section that starts with "you". Everywhere else says cluster admins or developers. Just wanted to check if it needed to be only admins or anything.

File: modules/red-hat-marketplace-features.adoc [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+During the link:https://marketplace.redhat.com/en-us/documentation/clusters[cluster connection process],
+a Marketplace Operator is installed that updates the image registry secret, manages the catalog, and reports application usage.
+
+[id="marketplace-install-applications_{context}"]
+== Install applications
+
+Cluster administrators can link:https://marketplace.redhat.com/en-us/documentation/operators[install Marketplace applications] from within OperatorHub in {product-title}, or at the link:https://marketplace.redhat.com[Marketplace web application].
+
+Access installed applications from the web console under *Installed Operators*.
```
Comment (bergerhoffer) [inline modules/red-hat-marketplace-features.adoc:23]: * Access -> You can access
* Should we specify that it's the `*Operators* -> *Installed Operators* page`?

File: modules/red-hat-marketplace-features.adoc [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+== Deploy applications from different perspectives
+
+Deploy Marketplace applications from the web console's Administrator and Developer perspectives.
+
+[discrete]
+=== The Developer perspective
+
+Access newly installed capabilities from the Developer perspective’s catalog.
+
+For example, after a database Operator is installed, a developer can create an instance from the catalog within their Project. Database usage is aggregated and reported to the cluster administrator.
```
Comment (bergerhoffer) [inline modules/red-hat-marketplace-features.adoc:35]: Project -> project

File: modules/red-hat-marketplace-features.adoc [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+[id="marketplace-deploy_{context}"]
+== Deploy applications from different perspectives
+
+Deploy Marketplace applications from the web console's Administrator and Developer perspectives.
+
+[discrete]
+=== The Developer perspective
+
+Access newly installed capabilities from the Developer perspective’s catalog.
```
Comment (bergerhoffer) [inline modules/red-hat-marketplace-features.adoc:33]: Suggest being consistent between the wording of the two perspectives. Either start them both with "The <x> perspective..." or "You can <do xyz> from the <x> perspective"

File: modules/red-hat-marketplace-features.adoc [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+== Install applications
+
+Cluster administrators can link:https://marketplace.redhat.com/en-us/documentation/operators[install Marketplace applications] from within OperatorHub in {product-title}, or at the link:https://marketplace.redhat.com[Marketplace web application].
+
+Access installed applications from the web console under *Installed Operators*.
+
+[id="marketplace-deploy_{context}"]
+== Deploy applications from different perspectives
+
+Deploy Marketplace applications from the web console's Administrator and Developer perspectives.
```
Comment (bergerhoffer) [inline modules/red-hat-marketplace-features.adoc:28]: Deploy -> You can deploy

File: modules/red-hat-marketplace-features.adoc [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+For example, after a database Operator is installed, a developer can create an instance from the catalog within their Project. Database usage is aggregated and reported to the cluster administrator.
+
+This perspective does not include Operator installation and application usage tracking.
+
+[discrete]
+=== The Administrator perspective
+
+The Administrator perspective includes information about Operator installation and application usage.
+
+{product-title} administrators can launch application instances by browsing custom resource definitions (CRDs) within the *Installed Operators* list.
```
Comment (bergerhoffer) [inline modules/red-hat-marketplace-features.adoc:44]: Should this just say "Cluster administrators" like we say elsewhere in this module?

File: _topic_map.yml [hunk 9]
```diff
@@ -866,6 +866,9 @@ Topics:
 - Name: Pruning objects to reclaim resources
   File: pruning-objects
   Distros: openshift-origin,openshift-enterprise,openshift-webscale
+- Name: The Red Hat Marketplace
```
Comment (bergerhoffer) [inline _topic_map.yml:869]: Probably should name it something like "Using the Red Hat Marketplace", to be consistent with our other heading namings.

File: applications/red-hat-marketplace.adoc [hunk 10]
```diff
@@ -0,0 +1,10 @@
+[id="red-hat-marketplace"]
+= The Red Hat Marketplace
```
Comment (bergerhoffer) [inline applications/red-hat-marketplace.adoc:2]: Same comment for the heading title.

File: modules/red-hat-marketplace-features.adoc [hunk 11]
```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * applications/red-hat-marketplace.adoc
+
+[id="red-hat-marketplace-features_{context}"]
+= Red Hat Marketplace features
+
+Cluster administrators can use link:https://marketplace.redhat.com/en-us/documentation/getting-started[Red Hat Marketplace] to manage software on {product-title}, give developers self-service access to deploy application instances, and correlate application usage against a quota.
```
Comment (bergerhoffer) [inline modules/red-hat-marketplace-features.adoc:8]: Add "the" in front of the marketplace link

File: modules/red-hat-marketplace-features.adoc [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+
+You can install a common set of applications on {product-title} clusters that you connect to the Marketplace. You can also use the Marketplace to track cluster usage against your subscriptions or quota. Users that you add by using the Marketplace have their product usage tracked and billed to your organization.
+
+During the link:https://marketplace.redhat.com/en-us/documentation/clusters[cluster connection process],
+a Marketplace Operator is installed that updates the image registry secret, manages the catalog, and reports application usage.
+
+[id="marketplace-install-applications_{context}"]
+== Install applications
+
+Cluster administrators can link:https://marketplace.redhat.com/en-us/documentation/operators[install Marketplace applications] from within OperatorHub in {product-title}, or at the link:https://marketplace.redhat.com[Marketplace web application].
```
Comment (bergerhoffer) [inline modules/red-hat-marketplace-features.adoc:21]: or at the -> or from the

**Review by bergerhoffer — COMMENTED** [review]
Just a few suggestions.

File: modules/red-hat-marketplace-features.adoc (hunk 3 above)
Comment (maxwelldb) [reply modules/red-hat-marketplace-features.adoc:13]: @robszumski I'd guess that this is admins only. Am I right?

File: applications/red-hat-marketplace.adoc (hunk 2 above)
Comment (robszumski) [reply applications/red-hat-marketplace.adoc:8]: whoops that might have snuck in during a copy/paste from the Marketplace team

File: modules/red-hat-marketplace-features.adoc (hunk 3 above)
Comment (robszumski) [reply modules/red-hat-marketplace-features.adoc:13]: yes, this is admins

**Comment by maxwelldb** [comment]
@robszumski How does this after incorporating https://github.com/openshift/openshift-docs/pull/21179/commits/fe16a762cab16f04f319677da44a15bcbf61b51a look to you?

**Comment by robszumski** [comment]
@maxwelldb LGTM

**Review by maxwelldb — APPROVED** [review]

**Comment by maxwelldb** [comment]
Trying out squash + merge GUI. Don't panic, onlookers. :)

**Pull request merged**